website/www/site/i18n/home/calendar/en.yaml

```diff
@@ -0,0 +1,42 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+ 
+- id: home-calendar-title
+  translation: "Stay up to date with Beam"
+- id: home-calendar-category-tag-first
+  translation: "Category"
+- id: home-calendar-category-date-first
+  translation: "2020/02/21"
+- id: home-calendar-card-title-first
+  translation: "Performance-Driven Runtime Type Checking for the Python SDK"
```

Comment:
In the future, will this data be pulled from recent blog posts, or will we have to keep it up to date?

Reply:
Great that you asked. Yeah, in the future, we'll have a separate stage for functionality improvement (the current stage is about design implementation). And the event will be synced with the event page (which will be implemented later).